app/lib/request.ts

```diff
@@ -37,7 +37,7 @@ export async function requestCredential(credentialRequestParams: CredentialReque
 
   const { accessToken } = await authorize(config).catch((err) => {
     console.error(err);
-    throw Error('Unable to receive credential');
+    throw Error('Unable to receive credential:\nAuthorization failed');
```

Comment:
```suggestion
    throw Error('Unable to receive credential: Authorization with the issuer failed');
```
No `\n` chars in error messages please.